native/cocos/bindings/jswrapper/v8/ScriptEngine.cpp

```diff
@@ -115,11 +116,12 @@ ccstd::string stackTraceToString(v8::Local<v8::StackTrace> stack) {
         stackStr += tmp;
         stackStr += "]";
         stackStr += (funcName.empty() ? "anonymous" : funcName.c_str());
-        stackStr += "@";
```

Comment:
Why remove `@` here?

Reply:
The format is adjusted according the Devtool output format

Reply:
Got it.